app/scripts/views/sign_in.js

```diff
@@ -310,4 +325,11 @@ function (Cocktail, p, BaseView, FormView, SignInTemplate, Session,
   );
 
   return View;
+
+  function interTabSignInHandler (event) {
+    if (this.broker.hasCapability('interTabSignIn')) {
+      this.user.setSignedInAccount(event.data)
+        .then(this.navigate.bind(this, 'settings'));
```

Comment:
`settings` seems reasonable here, though if all brokers redirect to the `/settings` page, the capability check might not be necessary. In [the comment here](https://github.com/mozilla/fxa-content-server/pull/3144/files#diff-e28aae2c15d692d7632811c383025a76R65), you mention that OAuth users should not redirect to the relier as the reason for the capability. If the user is always redirected to settings, they are never redirected to the relier AFAICT.